src/ui/linkballoonpanel.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Model from '../../ui/model.js';
+import Button from '../../ui/button/button.js';
+import ButtonView from '../../ui/button/buttonview.js';
+import BalloonPanel from '../../ui/balloonpanel/balloonpanel.js';
+import LabeledInput from '../../ui/labeledinput/labeledinput.js';
+import LabeledInputView from '../../ui/labeledinput/labeledinputview.js';
+import LinkForm from './linkform.js';
+import LinkFormView from './linkformview.js';
+import InputText from '../../ui/inputtext/inputtext.js';
+import InputTextView from '../../ui/inputtext/inputtextview.js';
+
+/**
+ * The link balloon panel controller class.
+ *
+ * 		const model = new Model( {
+ *			maxWidth: 300,
+ *			url: 'http://ckeditor.com'
+ *		} );
+ *
+ *		// An instance of LinkBalloonPanel.
+ *		new LinkBalloonPanel( model, new LinkBalloonPanelView() );
+ *
+ * See {@link link.ui.LinkBalloonPanelView}.
+ *
+ * @memberOf link.ui
```

Comment:
But it's defined inside `link/ui/`. Then the namespace should reflect this, because this is how you'll import those classes.